script/tool/README.md

```diff
@@ -42,31 +38,36 @@ command is targetting. An package name can be any of:
 - A combination federated_plugin_name/package_name (e.g.,
   `path_provider/path_provider` for the app-facing package).
 
+The examples below assume they are being run from the repository root, but
+the script works from anywhere. If you develop in flutter/packages frequently,
```

Comment:
anywhere within packages?

Reply:
Anywhere! `--current-package` only works in the packages directory, but with this change if you want to run `dart run /absolute/path/to/the/tool some-command --package=a_package` from some random other location it'll work.